components/ws-manager/pkg/manager/testdata/cpwp_custom_storage.golden

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
Just curious on why this file path with prefix `cpwp`? What is `cpwp` means?

I ask because the other files are with prefix `cdwp`, also I don't know what's the `cdwp` means.

Reply:
yup